app/client/components/contactUs/contactUsForm.tsx

```diff
@@ -285,25 +277,23 @@ export const ContactUsForm = (props: ContactUsFormProps) => {
             formValidationState.inValidationMode &&
             !formValidationState.email.isValid
           }
-          errorMessage={formValidationState.email.message}
+          errorMessage={formValidationState.email.errorMessage}
         />
-        {props.editableSubjectLine ? (
+        {props.editableSubject ? (
           <Input
             label="Subject of enquiry"
             type="text"
-            width={50}
-            changeSetState={newSubjectLine =>
-              setSubjectLine(newSubjectLine.substr(0, 50))
-            }
-            value={subjectLine}
+            width={100}
```

Comment:
This change has changed not only the character limit but the max-width of the input field, resulting in the following on desktop breakpoints:
![Screenshot 2020-11-20 at 13 35 24](https://user-images.githubusercontent.com/2510683/99806294-ba12fe00-2b35-11eb-978f-380e9b099e8f.png)